scripts/src/java/org/oppia/android/scripts/RepoFile.kt

```diff
@@ -0,0 +1,61 @@
+package org.oppia.android.scripts
+
+import java.io.File
+
+/** Helper class which contains all the file collection helper methods */
+class RepoFile() {
+  companion object {
+    /**
+     * Collects the paths of all the files which are needed to be checked.
+     *
+     * @param repoPath the path of the repo
+     * @param allowedDirectories a list of all the directories which needs to be checked
+     * @return all files which needs to be checked
+     */
+    fun collectSearchFiles(
+      repoPath: String,
+      allowedDirectories: List<String>,
+      allowedExtension: String = "",
+      exemptionsList: List<String> = listOf()
+    ):
+      List<File> {
```

Comment:
keep it in the same line above, and could you please set up macro and use it as mentioned in my earlier reviews?

Reply:
Done.

Reply:
I ran all the kotlin files under `ktlint` https://github.com/oppia/oppia-android/wiki/Ktlint-Guide#commands formatting check. It is giving no errors.